<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<bitbot>
  <logger path="/home/dknt/Project/bitbot-gz-ws/log/" level="info"/>
  <backend port="12888" settings_file="./backend.json"/>
  <bus>
    <device id="0" type="GzJoint" name="Rarm_shoulder_p_joint" 
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="1" type="GzJoint" name="Larm_shoulder_p_joint" 
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="3" type="GzImu" name="imu"/>

    <device id="4" type="GzJoint" name="Rleg_hip_p_joint"
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="5" type="GzJoint" name="Rleg_hip_r_joint"
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="6" type="GzJoint" name="Rleg_hip_y_joint"
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="7" type="GzJoint" name="Rleg_knee_joint"
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="8" type="GzJoint" name="Rleg_ankle_p_joint"
            mode="torque" p_gain='30' d_gain='1' i_gain='0'/>
    <device id="9" type="GzJoint" name="Rleg_ankle_r_joint"
            mode="torque" p_gain='30' d_gain='1' i_gain='0'/>

    <device id="10" type="GzJoint" name="Lleg_hip_p_joint"
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="11" type="GzJoint" name="Lleg_hip_r_joint"
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="12" type="GzJoint" name="Lleg_hip_y_joint"
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="13" type="GzJoint" name="Lleg_knee_joint"
            mode="torque" p_gain='100' d_gain='10' i_gain='0'/>
    <device id="14" type="GzJoint" name="Lleg_ankle_p_joint"
            mode="torque" p_gain='30' d_gain='1' i_gain='0'/>
    <device id="15" type="GzJoint" name="Lleg_ankle_r_joint"
            mode="torque" p_gain='30' d_gain='1' i_gain='0'/>

  </bus>
</bitbot>
